src/analyses/base.ml

```diff
@@ -9,13 +9,15 @@ module Q = Queries
 
 module GU = Goblintutil
 module ID = ValueDomain.ID
+module IDB = IntDomain.IntDomBranchingTuple
 module IdxDom = ValueDomain.IndexDomain
 module IntSet = SetDomain.Make (IntDomain.Integers)
 module AD = ValueDomain.AD
 module Addr = ValueDomain.Addr
 module Offs = ValueDomain.Offs
 module LF = LibraryFunctions
 module CArrays = ValueDomain.CArrays
+module DefExc = IntDomain.DefExc
```

Comment:
unused?

Reply:
Done, in e2c3e47715c9c3d4879a9d5dea1b583e3f767677.